web/public/js/directives/cd-award-badge/index.js

```diff
@@ -0,0 +1,50 @@
+;(function () {
+  'use strict';
+
+  angular
+    .module('cpZenPlatform')
+    .component('cdAwardBadge', {
+      bindings: {
+        user: '<'
+      },
+      controller: ['cdBadgesService', 'usSpinnerService', '$translate', 'alertService', '$rootScope', function (cdBadgesService, usSpinnerService, $translate, alertService, $rootScope) {
+        var ctrl = this;
+        ctrl.evidence = '';
+
+        ctrl.badgeSelection = ctrl.badgeSelection || {};
```

Comment:
Possible leftover from previous implementation, will check and probably get rid of it.